### PR TITLE
feat: use new api to get the workspace member role

### PR DESCRIPTION
**Pull request by speed2exe**

### CODE REVIEW

**Comment by LucasXu0** [comment]
@speed2exe The Rust CI failed. After checking the integration test error, I found that it was unable to create a new page in local mode.

File: frontend/rust-lib/flowy-ai/src/local_ai/watch.rs [hunk 1]
```diff
@@ -5,13 +5,15 @@ use tokio::sync::mpsc::{unbounded_channel, UnboundedReceiver};
 use tracing::{error, trace};
 
 #[cfg(any(target_os = "windows", target_os = "macos", target_os = "linux"))]
+#[allow(dead_code)]
 pub struct WatchContext {
   #[allow(dead_code)]
```
Comment (LucasXu0) [inline frontend/rust-lib/flowy-ai/src/local_ai/watch.rs:10]: Do we still need to keep `#[allow(dead_code)]` for the `watcher`?

Comment (speed2exe) [reply frontend/rust-lib/flowy-ai/src/local_ai/watch.rs:10]: we dont need this anymore, I'll remove it

**Comment by LucasXu0** [comment]
Fetch the member role from the workspace list instead of sending a request for each workspace to reduce API calls and UI loading time.

**Pull request merged**